src/classification/machine learning.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.naive_bayes import GaussianNB
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.metrics import confusion_matrix
+from sklearn.metrics import accuracy_score
+
+class MachineLearning():
+
+    def __init__(self):
+
+        print("Loading dataset ...")
+
+        self.flow_dataset = pd.read_csv('/content/drive/MyDrive/COMS6200/NF-UNSW-NB15.csv')
```

Comment:
Change it to pd.read_csv('../../data/NF-UNSW-NB15.csv') because we'll be adding the CSV folder inside it from the data folder.